### ISSUE TITLE
Request for remembering the cheat selections for last time.

### ISSUE THREAD

**Issue by CheatAbuser** [emukidid/swiss-gc#440]
The current version, r990, can only remember the cheat selections before starting games or rebooting GC. After rebooting, I have to select the cheats again. Is it possible to remember the cheat selections for last time? eg. I always use an Exp Multiplier cheat in an RPG. If Swiss can auto enable the Exp Multiplier cheat, that would be great!
The auto enable all cheats feature is not very useful. I have other cheats seldom used in the TXT files.

**Comment by Extrems** [comment]
Duplicate of issue #345.
